pkg/network/tracer/connection/dump.go

```diff
@@ -24,166 +24,163 @@ import (
 )
 
 //nolint:revive // TODO(NET) Fix revive linter
-func dumpMapsHandler(manager *manager.Manager, mapName string, currentMap *ebpf.Map) string {
-	var output strings.Builder
 
+func dumpMapsHandler(w io.Writer, manager *manager.Manager, mapName string, currentMap *ebpf.Map) {
 	switch mapName {
 
 	case "connectsock_ipv6": // maps/connectsock_ipv6 (BPF_MAP_TYPE_HASH), key C.__u64, value uintptr // C.void*
-		output.WriteString("Map: '" + mapName + "', key: 'C.__u64', value: 'uintptr // C.void*'\n")
+		io.WriteString(w, "Map: '"+mapName+"', key: 'C.__u64', value: 'uintptr // C.void*'\n")
 		iter := currentMap.Iterate()
 		var key uint64
 		var value uintptr // C.void*
 		for iter.Next(unsafe.Pointer(&key), unsafe.Pointer(&value)) {
-			output.WriteString(spew.Sdump(key, value))
+			io.WriteString(w, spew.Sdump(key, value))
```

Comment:
should this file use `spew.Fdump` like the other files?

Reply:
Yep, I missed that, changing it!